lessons/bn/basics/basics.md

```diff
@@ -213,9 +213,11 @@ iex> not 42
 ** (ArgumentError) argument error
 ```
 
-### কম্প্যারিজন
+বি:দ্র: এলিক্সির এর `and` এবং `or` আসলে আর্ল্যাং এর `andalso` এবং `orelse` অপারেটরে ম্যাপ করা।
+
+### তুলনা
 
-এলিক্সির গতানুগতিক সব কম্প্যারিজন অপারেটর সাপোর্ট করে : `==`, `!=`, `===`, `!==`, `<=`, `>=`, `<` এবং `>`.
+এলিক্সির গতানুগতিক সব কম্প্যারিজন(তুলনামূলক) অপারেটর সাপোর্ট করে : `==`, `!=`, `===`, `!==`, `<=`, `>=`, `<` এবং `>`.
```

Comment:
We can replace the `.` with Bengali full stop `।`

Reply:
fixed.